__tests__/integration/schema-version.test.ts

```diff
@@ -5,13 +5,17 @@ describe("schema version is returned by the client", () => {
   const client = getClient();
   it("returns the schema version", async () => {
     const resTs = await client.query(fql`
-      Collection.create({ name: "TestColl" })
+      if (Collection.byName("TestColl") == null) {
+        Collection.create({ name: "TestColl" })
+      } else {
+        TestColl.definition.update({})
```

Comment:
Would this work? To avoid the write?

```suggestion
        TestColl.definition
```